hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -414,7 +414,7 @@
   public static final String OZONE_CLIENT_RETRY_MAX_ATTEMPTS_KEY =
       "ozone.client.retry.max.attempts";
   public static final int OZONE_CLIENT_RETRY_MAX_ATTEMPTS_DEFAULT =
-      10;
+      30;
```

Comment:
Any reason for change in this value?

Reply:
I just thought we should try more number of times on each OM (10 on each OM for a 3 node OM cluster).